packages/web3-eth-abi/src/formatters.js

```diff
@@ -142,7 +142,11 @@ var signedIsNegative = function (value) {
  * @returns {BN} right-aligned output bytes formatted to big number
  */
 var formatOutputInt = function (param) {
-    var value = param.staticPart() || "0";
+    var value = param.staticPart();
```

Comment:
We have to make sure the this function in return nothing when there is an empty value returned. could you run some tests?

Reply:
I'm confused why there's this special case for `int`? `uint` for example does not have this default value. My tests cover all of them.